pkg/kudoctl/util/kudo/kudo.go

```diff
@@ -102,6 +91,20 @@ func NewClientFromK8s(kudo versioned.Interface, kube kubernetes.Interface) *Clie
 	return &result
 }
 
+func (c *Client) VerifyServedCRDs(kubeClient *kube.Client) error {
+	result := verifier.NewResult()
+	err := crd.NewInitializer().VerifyServedVersion(kubeClient, v1beta1.SchemeGroupVersion.Version, &result)
```

Comment:
The `v1beta1.SchemeGroupVersion.Version` parameter sort of hard-codes support for the `v1beta1` version. Once we introduce a new version, we _should not forget_ to update this method with the check with the new version?

Reply:
Yeah, that's true. Sadly there's no way to get the GV from the generated client - that would make it a bit easier.